package.json

```diff
@@ -44,7 +47,11 @@
   },
   "author": "",
   "babel": {
-    "presets": ["es2015", "stage-0", "react"],
+    "presets": [
+      "es2015",
+      "stage-0",
```

Comment:
@coopy Yes, `stage-0` is bleeding-edge transform, whereas you can kinda think of `es2015` as `stage-1-million`. ;)